e2e/suite/plan_create_test.go

```diff
@@ -1,6 +1,8 @@
 package suite_test
 
 import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
```

Comment:
In this file, you are adding these two imports but nothing else. I wonder if this is correct or a copy/paste?